docs/questions/query-builder/expressions/countif.md

```diff
@@ -0,0 +1,273 @@
+---
+title: CountIf
+---
+
+# CountIf
+
+`CountIf` counts the total (not unique) rows in a table that match a condition.
+
+Syntax: `CountIf(condition)`.
+
+Example: in the table below, `CountIf([Plan] = "Basic")` would return 3.
+
+| ID  | Plan        |
+|-----|-------------|
+| 1   | Basic       |
+| 2   | Basic       |
+| 3   | Basic       |
+| 4   | Business    |
+| 5   | Premium     |
+
+> [Aggregations](../expressions-list.md#aggregations) like `CountIf` should be added to the query builder's [**Summarize** menu](../../query-builder/introduction.md#summarizing-and-grouping-by) > **Custom Expression** (scroll down in the menu if needed).
+
+## Parameters
+
+`condition` is a [function](../expressions-list.md#functions) or [conditional statement](../expressions.md#conditional-operators) that returns a boolean value (`true` or `false`).
+
+## Multiple conditions
+
+| ID  | Plan        | Active Subscription |
+|-----|-------------| --------------------|
+| 1   | Basic       | true                |
+| 2   | Basic       | true                | 
+| 3   | Basic       | false               |
+| 4   | Business    | false               |
+| 5   | Premium     | true                |
+
+### Mandatory matches
```

Comment:
```suggestion
### Required conditions
```
Multiple and mandatory are both abstract words that start with M. This makes them a bit easier to distinguish.